src/routes/(infoPages)/[topTierPage]/+page.server.ts

```diff
@@ -254,8 +283,16 @@ export const load = async ({
                 : undefined,
             }
           : undefined,
-        relatedNews: taggedData?.newsCollection ?? { items: [] },
-        relatedEvents: taggedData?.eventEntryCollection ?? { items: [] },
+        relatedNews: taggedData?.newsCollection?.items.length
+          ? taggedData?.newsCollection
+          : matchedTopTier.recentNewsCollection?.items.length
+          ? matchedTopTier.recentNewsCollection
+          : { items: [] },
+        relatedEvents: taggedData?.eventEntryCollection?.items.length
+          ? taggedData?.eventEntryCollection
+          : matchedTopTier.upcomingEventsCollection?.items.length
+          ? matchedTopTier.upcomingEventsCollection
+          : { items: [] },
```

Comment:
LGTM, maybe just add a comment here that tagged news and events will get priority until we cutover to using the new fields.